docs/api/field_type_reference.md

```diff
@@ -1623,6 +1612,10 @@ Example response:
 #### Mutation
 
 To create a Matrix Field Type you need to define Field Type and Field definition identifiers.
+The types that are used for input and returned are named after the Type and the Field: 
+
+- `{TypeIdentifier}{FieldIdentifier}RowInput` e.g. `dish.nutritionFacts`, `event.agenda`: `DishNutritionFactsRowInput`, `EventAgendaRowInput`
+- `{TypeIdentifier}{FieldIdentifier}Row`
```

Comment:
This one should still be documented in the section about _retrieving_ matrix content with GraphQL ;)